### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by tcrasset** [Nooreyni/DL-wine-quality#1]
# Structured data review

## Good

- Nice idea of doing checkpoints for long training
- Good start of README
- Start of EDA, but you could have done more.
- Nice resampling
- Nice normalization
- Good use of `kerastuner`
- I like that you used a validation set

## To improve

- You should not upload all your checkpoints to the cloud, most of them are 1. unnecessary, and 2. way to heavy
- Loss is not decreasing in `pytorch` model.
- EDA could have been in separate file
- Your model overfits (`61.04%` on train, `31.9%` on test)
- You are testing way too many un-important values: You don't need to test whether 2,4,6,8,10,...,20 layers are best. Same goes for dense units. You should focus on testing different orders of magnitude (5, 10, 50, 100, 500 for example).
- There should not be a `.ipynb_checkpoints` folder, use a `.gitignore`.
